### ISSUE TITLE
Any plans to release for Linux?

### ISSUE THREAD

**Issue by vinRosso** [nimsb/Baguette#1]

**Comment by nimsb** [comment]
yes, and also on MacOS but I need to install the whole thing, and I've never installed Linux and software yet. I also need to figure how to compile it on both of those machine too ...

**Comment by vinRosso** [comment]
Great! Hit me up if you need some help with the Linux version.
